samples/matter/manufacturer_specific/src/app_task.cpp

```diff
@@ -50,19 +52,29 @@ void AppTask::UpdateNordicDevkitClusterState()
 
 		dk_read_buttons(&button_state, nullptr);
 
-		status = Clusters::NordicDevKit::Attributes::UserLED::Set(kNordicDevKitEndpointId,
-								Nrf::GetBoard().GetLED(Nrf::DeviceLeds::LED2).GetState());
+		status = Clusters::NordicDevKit::Attributes::UserLED::Set(
+			kNordicDevKitEndpointId, Nrf::GetBoard().GetLED(Nrf::DeviceLeds::LED2).GetState());
 
 		if (status != Protocols::InteractionModel::Status::Success) {
 			LOG_ERR("Updating NordicDevkit cluster failed: %x", to_underlying(status));
 		}
 
 		status = Clusters::NordicDevKit::Attributes::UserButton::Set(kNordicDevKitEndpointId,
-										    BUTTON2_MASK & button_state);
+									     BUTTON2_MASK & button_state);
```

Comment:
@kkasperczyk-no, @maciejbaczmanski it seems that you have different settings for the clang format. :)

Reply:
We have verified with @ArekBalysNordic that our clangs format the code the same. @maciejbaczmanski woudl you mind checking what are you settings?